napari/components/viewer_model.py

```diff
@@ -551,7 +551,7 @@ def add_image(
         shear=None,
         affine=None,
         opacity=1,
-        blending=None,
+        blending='translucent_no_depth',
```

Comment:
This change changes the `split_channels` behavior, because `split_channels` only assigns blending if blending was `none`. Blending when splitting channels used to be `additive` for all then I changed it to `translucent` for base then `additive` for rest to make it work in Light theme.
https://github.com/napari/napari/pull/4394
This PR makes it just `translucent_no_depth` for all channel layers. 
(Can be seen by running @jni example with this PR).
With the change at line 743 for the case where channel_axis is None, perhaps it's better to change the `split_channels` function to also use `translucent_no_depth`:
https://github.com/napari/napari/blob/37e3bcf7d8ac0405b671e7428ece5fca9d2d9b1a/napari/layers/utils/stack_utils.py#L84
If you manually change `nuclei` to `additive` in @jni example, the fixed behavior still works.